plugins/gozero/core/logx/logx_debug_intercepter.go

```diff
@@ -0,0 +1,30 @@
+package logx
+
+import (
+	"github.com/apache/skywalking-go/plugins/core/operator"
+	"github.com/apache/skywalking-go/plugins/core/tracing"
+)
+
+type LoggerDebugInterceptor struct {
+}
+
+// BeforeInvoke implements the instrument.Interceptor interface.
+func (h *LoggerDebugInterceptor) BeforeInvoke(invocation operator.Invocation) error {
+	if config.CollectLogx {
+		span, err := createLogxSpan(invocation, "logx.debug")
```

Comment:
We should consider another format of span name. Such as redis plugin, we use `redis/[operation]` as span name.